src/layer/AbstractLayer.ts

```diff
@@ -32,12 +32,12 @@ export class AbstractLayer {
     throw new Error('Not implemented');
   }
 
-  public findTiles(
+  public async findTiles(
     bbox: BBox, // eslint-disable-line @typescript-eslint/no-unused-vars
-    fromTime: any, // eslint-disable-line @typescript-eslint/no-unused-vars
-    toTime: any, // eslint-disable-line @typescript-eslint/no-unused-vars
-    maxCount?: number, // eslint-disable-line @typescript-eslint/no-unused-vars
-    offset?: number, // eslint-disable-line @typescript-eslint/no-unused-vars
+    fromTime: Date, // eslint-disable-line @typescript-eslint/no-unused-vars
+    toTime: Date, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
if `fromTime` and `toTime` in `Tile` are now `Date`s, the `findFlyoverIntervals()` shoould be changed as well.

Example: `tiles.sort((a, b) => (new Date(a.sensingTime).getTime() > new Date(b.sensingTime).getTime() ? 1 : -1));`, etc.

Reply:
Thank you, I have removed the Date constructors now.

While at it, I have also fixed sorting function (if equal times were compared, the result depended on the order of arguments).

Reply:
Thanks!
I closed the MR mentioned in the original comment.

Also thanks for fixing the sorting function, I have over-engineered it.